docs/source/aws/lambda.rst

```diff
@@ -43,7 +43,7 @@ policy filters, and apply the policy actions to matching resources.
 CloudTrail API Calls
 ++++++++++++++++++++
 
-Lambdas can receive CWE over CloudTrail API calls with delay of 90s at P99.
+Lambdas can receive CWE over CloudTrail API calls with seconds of delay at P99.
```

Comment:
I would call out what "CWE" stands for when it first occurs, which is line 32. That line would become

```
Custodian provides for policy level execution against any CloudWatch Events stream. Henceforth "CloudWatch Events" will be abbreviated as CWE.
```